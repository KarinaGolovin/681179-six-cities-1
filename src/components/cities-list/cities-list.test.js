import React from 'react';
import renderer from 'react-test-renderer';
import CitiesList from './cities-list.jsx';
import withActiveItem from '../../hocs/with-active-item/with-active-item';

const CitiesListWrapped = withActiveItem(CitiesList);

it(`CitiesList correctly renders`, () => {
  const component = renderer.create(
      <CitiesListWrapped
        cities={[`Test`, `Test Test`]}
        onCityClick={() => {}}
        onActiveItemChange={() => {}}
      />
  ).toJSON();

  expect(component).toMatchSnapshot();
});
