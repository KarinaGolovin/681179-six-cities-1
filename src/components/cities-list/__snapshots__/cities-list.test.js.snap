// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesList correctly renders 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item null"
      href="#"
      onClick={[Function]}
    >
      <span>
        Test
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item null"
      href="#"
      onClick={[Function]}
    >
      <span>
        Test Test
      </span>
    </a>
  </li>
</ul>
`;
